src/Media_Command.php

```diff
@@ -414,6 +417,9 @@ public function import( $args, $assoc_args = array() ) {
 
 			if ( Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
 				WP_CLI::line( $success );
+			} elseif ( Utils\get_flag_value( $assoc_args, 'porcelain_url') ) {
+				$file_location = wp_get_original_image_url( $success );
```

Comment:
`wp_get_original_image_url()` only works for images, not other types of media. Can we just use `wp_get_attachment_url()`?

Reply:
Note that this is also why the tests are currently failing on WP 3.7, as `wp_get_original_image_url` was only introduced in 5.3. Using `wp_get_attachment_url()` instead would solve this.

Reply:
Ah, good point. `wp_get_attachment_url()` was returning the `-scaled` version of the image. Will dig in a bit.